blueprints/ember-cli-uniq/index.js

```diff
@@ -2,9 +2,9 @@
 'use strict';
 
 module.exports = {
-  normalizeEntityName: function() {}, // no-op since we're just adding dependencies
```

Comment:
I'm not sure about how this change impacts the addon, since this is nodejs

Reply:
this is our eslinter doing

Reply:
🎩 🐇